djangocms_versioning/templates/djangocms_versioning/admin/archive_confirmation.html

```diff
@@ -19,7 +19,7 @@ <h4>{% blocktrans %} Version number: {{ version_number }}{% endblocktrans %}</h4
     {% csrf_token %}
     <input  class="button confirm-link js-versioning-keep-sideframe"
             type="submit"
-            value="{% trans 'Yes, I\'m sure' %}">
+            value="{% trans "Yes, I\'m sure" %}">
```

Comment:
We might be better off using `translate` as the tagname.

`trans` still exists as an alias in django main, but I think it'll get removed at some point.

Reply:
True. I've replaced all `{% trans %}` tags with `{% translate %}`